docs/day2/04_multiqc.md

```diff
@@ -166,13 +252,80 @@ nextflow run main.nf -resume
 ```console title="Output"
 Launching `main.nf` [hopeful_swanson] DSL2 - revision: a4304bbe73
 
-[aa/3b8821] INDEX                          [100%] 1 of 1, cached: 1 ✔
-[c2/baa069] QUANTIFICATION (salmon on gut) [100%] 1 of 1, cached: 1 ✔
-[ad/e49b20] FASTQC (FASTQC on gut)         [100%] 1 of 1, cached: 1 ✔
-[a3/1f885c] MULTIQC                        [100%] 1 of 1 ✔
+[aa/3b8821] INDEX          [100%] 1 of 1, cached: 1 ✔
+[c2/baa069] QUANTIFICATION [100%] 1 of 1, cached: 1 ✔
+[ad/e49b20] FASTQC         [100%] 1 of 1, cached: 1 ✔
+[a3/1f885c] MULTIQC        [100%] 1 of 1 ✔
 
 ```
 
 > Inspect `results/multiqc_report.html`, maybe Poll something in the file  
 
 You have a working pipeline for a single paired-end sample!
+
+## 2.4.3 Adding a `tag` directive
```

Comment:
Thinking of moving this back to the beginning of the next section (2.5), and having the `-with-report`/`-with-trace` etc. exercises as advanced to reduce time

Reply:
Perhaps leave this here.